app/src/main/java/edu/uw/myuw_android/CommonWebViewFragment.kt

```diff
@@ -104,8 +121,21 @@ class CommonWebViewFragment: Fragment() {
     override fun onStart() {
         super.onStart()
         (activity as AppCompatActivity).supportActionBar!!.title = ""
+        authorizationService = AuthorizationService(activity!!)
```

Comment:
Same question as above for supportActionBar here.
And just a side note, we can substitute activity!! with requireActivity() which throws an exception when it's null. And for the context!! below, we can use requireContext().